### ISSUE TITLE
Please continue good work!

### ISSUE THREAD

**Issue by haxworx** [LeBlue/enlightenment-arc-theme#1]
There is no issue, just would like to say this is nice!!! Keep going!!!

**Comment by NuLogicSystems** [comment]
Yes, please. This is the best theme I've seen for E so far.

**Comment by LeBlue** [comment]
Yes, I am working an it and will continue. But my time for working on this project is limited and there are a lot of enlightenment and efl widgets, enlightenment modules still missing (see README). Some widgets are rarely used and most modules require a lot of work. I do not really want to port unused parts, so it would be helpful if you (or anybody) has any suggestions, what to do next.

Yes, some modules, like mixer, econnman, are on my list as well as the 3rd variant, arc-darker. But as I said: a lot or work and SUGGESTIONS!?
